pkg/resources/password.go

```diff
@@ -1,7 +1,11 @@
 package resources
 
 import (
-	"math/rand"
+	"crypto/rand"
+	"encoding/binary"
+	"log"
+	"math/big"
+	rand2 "math/rand"
```

Comment:
Very minor suggestion but we could opt for a more descriptive name like `mathrand` perhaps

Reply:
Everything else looks good so happy to lgtm if you don't think it's worth making this change

Reply:
Don't have a strong opinion on this. The `rand2` was given by the IDE :thinking: not a big deal - renamed and retesting